### DISCUSSION TITLE
Scoped class font style

### DISCUSSION THREAD

**Discussion by antoniosarosi** [vuejs/language-tools#1653] [Q&A]
Volar seems to underline classes applied with scoped style. Here's an example:

![image](https://user-images.githubusercontent.com/54243067/181734810-6eb1a999-d858-4291-9859-e14437258f5f.png)

I usually set `"editor.lineHeight"` to a value that makes the line height big enough so that there's no overlapping, but I don't like having huge spaces between lines. If there are underlines, the line height has to be increased otherwise there's overlapping. Is it possible to change this settings? For example, italic instead of underline. I have tried this in `jsconfig.json`:

```json
{
    "vueCompilerOptions": {
        "experimentalResolveStyleCssClasses": "never"
    }
}
```

This works, there's no underline, but it seems to disable intellisense inside template, there's a warning on the template tag: `TypeScript intellisense is disabled`. If I add `"compilerOptions": { "jsx": "preserve" }` then there's intellisense again but also underlines.

**Comment by johnsoncodehk** [comment]
Yes it could be change, you can open a feature request.
